static/css/main.css

```diff
@@ -7,68 +7,127 @@ body {
   font-size: 16px;
   font-family: 'Lato', sans-serif; }
 
-.container {
-  width: 100%; }
+.encloser {
+  display: flex;
+  min-height: 100vh;
```

Comment:
Hrm, good catch. I'm not sure whether or not this is good form from a flexbox point of view, but it's actually unnecessary. Given the layout as it is, it will fill the viewport top to bottom without this. I'll remove.